papyrus/xsd.py

```diff
@@ -67,7 +67,7 @@ def add_column_xsd(self, tb, column, attrs):
         if column.nullable:
             attrs['minOccurs'] = str(0)
             attrs['nillable'] = 'true'
-        for cls, xsd_type in self.SIMPLE_XSD_TYPES.iteritems():
+        for cls, xsd_type in self.SIMPLE_XSD_TYPES.items():
```

Comment:
OK, I will leave it like this. Since this syntax works in both 2 and 3 I think it is cleaner to use it.